dso-l2/src/main/java/com/tc/l2/state/StateManagerImpl.java

```diff
@@ -200,19 +203,23 @@ private void runElection() {
     if (!electionStarted()) {
       return;
     }
+
+    // This topology will be used throughout the election process
+    Topology topology = TopologyManager.get().getTopology();
+
```

Comment:
can this be passed in from somewhere.  Singletons don't look right here.

Reply:
changed now